lib/publiccloud/provider.pm

```diff
@@ -256,7 +269,17 @@ sub terraform_apply {
         $image, $args{count}, $instance_type, $self->region);
 
     assert_script_run($cmd);
-    assert_script_run('terraform apply myplan', TERRAFORM_TIMEOUT);
+    my $ret = script_run('terraform apply myplan', TERRAFORM_TIMEOUT);
+    unless (defined $ret) {
+        type_string(qq(\c\\));        # Send QUIT signal
+        sleep 1;                      # give some time to Quit
```

Comment:
predicting that we will need to increase this value over a time ... any chance to replace it with `wait_serial` somehow ? Main question for what you could wait here ...